growth-program/src/rewards/withdraw.rs

```diff
@@ -60,9 +80,6 @@ pub trait WithdrawRewardsModule:
     fn finish_program(&self, project_id: ProjectId) {
         self.require_not_paused();
 
-        let caller = self.blockchain().get_caller();
```

Comment:
Maybe add a check if this endpoint was already called? To avoid allowing users to call it multiple times.

Reply:
It would fail regardless at `rewards_info.take()`.